cfn.template

```diff
@@ -147,39 +147,51 @@ Resources:
                     regions=$(aws ec2 describe-regions --output text --query 'Regions[].RegionName')
                   fi
 
+                - # Deal with either "template" or "templates"
+                - |
+                  if [ -n "$template" ]; then
```

Comment:
this is probably fine but it gives precedence to the old naming convention? do you think it would be better to check for templates being empty? in reality it'd be a really weird scenario where both template and templates are set but I'm just thinking in terms of the code semantics

Reply:
Euch. Fair point.